nikola/plugin_categories.py

```diff
@@ -471,3 +472,221 @@ def import_file(self):
     def save_post(self):
         """Save a post to disk."""
         raise NotImplementedError()
+
+
+class Taxonomy(BasePlugin):
+    """Taxonomy for posts.
+
+    A taxonomy plugin allows to classify posts (see #2107) by
+    classification strings.
+    """
+
+    name = "dummy_taxonomy"
+
+    # Adjust the following values in your plugin!
+
+    # The classification name to be used for path handlers.
+    classification_name = "taxonomy"
```

Comment:
Yep, creating a taxonomy is not as simple as creating another plugin ;)

I just tested the separate docstrings with standard Python's `help()`, there they aren't displayed. So I guess I'll rewrite it into one big docstring at the beginning of the class.